pkg/vm/context.go

```diff
@@ -278,22 +278,26 @@ func (c *Context) IsDeployed() bool {
 }
 
 // DumpStaticSlot returns json formatted representation of the given slot.
+// Deprecated: to be removed in next version.
```

Comment:
Here and below, please, add a note that `(*Context).*Slot` method should be used to retrieve slot. Also please add an entry to ROADMAP.md (https://github.com/nspcc-dev/neo-go/blob/master/ROADMAP.md#deprecated-functionality) following the document style. Let's schedule deprecation of these methods to 0.108.0 release.